components/CommunityList/CommunityList.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { slugify } from 'utils';
+import { DefaultCard, Box, Loader } from 'ui-kit';
+import { CustomLink } from 'components';
+
+function CommunityList(props = {}) {
+  if (props.loading) return <Loader text="Loading" />;
+
+  const noData = props.data.length === 0;
+  if (noData)
+    return <Box as="p">You do not have any items to display right now.</Box>;
+
+  const featuredItems = props.data.filter(
+    item => item.featured && item?.coverImage?.sources[0]?.uri
+  );
+  const notFeaturedItems = props.data.filter(
+    item => !item.featured && item?.coverImage?.sources[0]?.uri
+  );
```

Comment:
I personally think this is fine, but we could do something like:

```js
const byFlag = flag => item => flag && item?.coverImage?.sources[0]?.uri;
const featuredItems = props.data.filter(byFlag(item => item.featured));
const notFeaturedItems = props.data.filter(byFlag(item => !item.featured));
```

Not sure if it's necessary, though 🤷‍♂️